conda_env/specs/__init__.py

```diff
@@ -2,13 +2,15 @@
 from .yaml_file import YamlFileSpec
 from .notebook import NotebookSpec
 from .requirements import RequirementsSpec
+from .jinja2_file import Jinja2Spec
 from ..exceptions import SpecNotFound
 
 all_specs = [
     BinstarSpec,
     NotebookSpec,
     YamlFileSpec,
-    RequirementsSpec
+    RequirementsSpec,
+    Jinja2Spec
```

Comment:
If the file is pure yaml, yes, but if it has jinja markers, `YamlFileSpec.can_handle` should return `False`.